packages/create-workleap/package.json

```diff
@@ -2,17 +2,33 @@
   "name": "create-workleap",
   "version": "0.0.0",
   "description": "Foundry-CLI Workleap project starter kit",
-  "bin": "./dist/src/bin.js",
+  "license": "Apache-2.0",
+  "repository": {
+      "type": "git",
+      "url": "git+https://github.com/workleap/wl-foundry-cli.git",
+      "directory": "packages/create-workleap"
+  },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
+  },
+  "type": "module",
+  "bin": "./dist/bin.js",
   "scripts": {
-    "build": "tsc",
+    "dev": "pnpm run build:dev && pnpm link --global",
+    "build": "tsup",
+    "build:dev": "tsup --sourcemap",
     "clean": "pnpm dlx rimraf dist coverage",
-    "lint:types": "tsc --noEmit",
-    "test": "jest",
-    "coverage": "jest --coverage"
+    "lint:types": "tsc --noEmit"
   },
   "dependencies": {
-    "@clack/core": "^0.3.2",
-    "@clack/prompts": "^0.6.3",
-    "picocolors": "^1.0.0"
+    "@clack/core": "0.3.2",
+    "@clack/prompts": "0.6.3",
+    "picocolors": "1.0.0"
+  },
+  "tsup": {
+    "entry": ["src/bin.ts"],
+    "clean": true,
+    "format": ["esm"]
```

Comment:
Fine for this PR but out of curiosity, will we be able to move `tsup` config to a `web-configs` package?

Reply:
Their configuration files can be .js so we can import a custom function that sets what we need or spread a config object